test/examples/Id-Strings.ju.golden

```diff
@@ -0,0 +1 @@
+[Function' (Func' (Like' {functionLikedName = string, functionLikeArgs = [], functionLikeBody = Body' (Primitive' (Prim' (Michelson :| [string]) ()) ()) (), annLike = ()}) ()) (),Function' (Func' (Like' {functionLikedName = pair, functionLikeArgs = [], functionLikeBody = Body' (Primitive' (Prim' (Michelson :| [pair]) ()) ()) (), annLike = ()}) ()) (),Function' (Func' (Like' {functionLikedName = unit, functionLikeArgs = [], functionLikeBody = Body' (Primitive' (Prim' (Param :| [unit]) ()) ()) (), annLike = ()}) ()) (),Function' (Func' (Like' {functionLikedName = storage, functionLikeArgs = [], functionLikeBody = Body' (Name' (unit :| []) ()) (), annLike = ()}) ()) (),Signature' (Sig' {signatureName = default, signatureUsage = Nothing, signatureArrowType = Infix' (Inf' {infixLeft = Name' (string :| []) (), infixOp = -> :| [], infixRight = Infix' (Inf' {infixLeft = Name' (storage :| []) (), infixOp = -> :| [], infixRight = Name' (storage :| []) (), annInf = ()}) (), annInf = ()}) (), signatureConstraints = [], annSig = ()}) (),Function' (Func' (Like' {functionLikedName = default, functionLikeArgs = [ConcreteA' (MatchLogic' {matchLogicContents = MatchName' para (), matchLogicNamed = Nothing, annMatchLogic = ()}) (),ConcreteA' (MatchLogic' {matchLogicContents = MatchName' storage (), matchLogicNamed = Nothing, annMatchLogic = ()}) ()], functionLikeBody = Body' (Application' (App' {applicationName = Name' (pair :| []) (), applicationArgs = List' (ListLit' [] ()) () :| [Name' (storage :| []) ()], annApp = ()}) ()) (), annLike = ()}) ()) (),Signature' (Sig' {signatureName = init-storage, signatureUsage = Nothing, signatureArrowType = Name' (storage :| []) (), signatureConstraints = [], annSig = ()}) (),Function' (Func' (Like' {functionLikedName = init-storage, functionLikeArgs = [], functionLikeBody = Body' (Name' (unit :| []) ()) (), annLike = ()}) ()) ()]
```

Comment:
Can we run a formatter on this, so it's not one big line?

Will that mess up how the golden tests work?

Reply:
I will do more testings and see if there is a way. I fear that if we change the golden file in anyway the tests would fail. But I'll see what I can do.

Reply:
If it just parses it as a haskell string there is a way to trick it, probably the same thing I've done in the FrontendTests (adding \ to the end and start to signal haskell where to begin again

Reply:
I'll format the golden files in another PR. Just really want this to merge first. All tests pass now.